README.md

```diff
@@ -49,10 +99,27 @@ Once you have opened Neo4j desktop, use the `New` button in the upper-left regio
 
 The graph will initially be empty, but once you have successfully loaded data, Neo4j Desktop provides an interface for exploring and visualizing relationships within the graph. To access it, click the blue "Open" button. The prompt at the top of this window processes [Cypher queries](https://neo4j.com/docs/cypher-refcard/current/); to start, try `MATCH (n:Statement {id:"civic.eid:1409"}) RETURN n`. Buttons on the left-hand edge of the results pane let you select graph, tabular, or textual output.
 
-
 ### Setting up normalizers
 
-The MetaKB calls a number of normalizer libraries to transform resource data and resolve incoming search queries. These will be installed as part of the package requirements, but require additional setup.
+The MetaKB calls a number of normalizer libraries to transform resource data and resolve incoming search queries. These will be installed as part of the package requirements, but may require additional setup. Optionally, you can point directly to deployed instances of the normalizers and bypass this setup if desired.
+
+#### Simple setup (recommended)
+
+If you would like to simply use the dev normalizers, use the following environment variables:
```

Comment:
probably a better way to do this or write it. I didn't include envs or tables for prod. We could probably make flags or something to do this for us at some point or in some way? I wasn't sure what to do for now. Kinda glossed over it since I was mostly focused on restructuring the repo in this PR

Reply:
We can remove this section since it really only applies to Wagner Lab (probably best to have in an internal document). Once I finish Dockerizing the normalizers, it will replace this section.